src/theme/theme.ts

```diff
@@ -100,27 +100,26 @@ export const theme: Theme = {
       fontWeight: "medium",
       lineHeight: 20 / 12, // 20px
     },
+    button1: {
+      fontFamily: "body",
+      fontSize: "base",
+      fontWeight: "semibold",
+      lineHeight: "body",
+    },
   },
 
   buttons: {
-    primary: {
-      color: "background",
-      bg: "primary",
-      "&:hover": {
-        opacity: 0.7,
-      },
+    filled: {
       cursor: "pointer",
-      transition: "all .3s linear",
+      transition: "all .2s linear",
```

Comment:
Any reason for these few styles to be here instead of with the rest of the css inside the component?

Reply:
Yes, the .3s creates a little slow animation, despite the number might suggest too little of a change, actually, it improves a lot the overall experience while the animation happens